### PR TITLE
Jinghan/unify serialize helper functions

### PR DESCRIPTION
**Pull request by jinghancc**
This PR unifies serialization helper functions for all backend types.

### CODE REVIEW

**Review by wfxr — APPROVED** [review]
LGTM

File: internal/database/dbutil/deserialize.go [hunk 1]
```diff
@@ -23,8 +24,13 @@ func DeserializeByValueType(i interface{}, valueType types.ValueType, backend ty
 		deserializer = dynamoDeserializer
 	case types.BackendSnowflake:
 		deserializer = snowflakeDeserializer
-	default:
+	case types.BackendRedis, types.BackendTiKV:
+		deserializer = kvDeserializer
+	case types.BackendMySQL, types.BackendPostgres, types.BackendSQLite,
+		types.BackendBigQuery, types.BackendRedshift:
 		deserializer = defaultDeserializer
```
Comment (wfxr) [inline internal/database/dbutil/deserialize.go:31]: defaultDeserializer => rdbDeserializer ?

Comment (jinghancc) [reply internal/database/dbutil/deserialize.go:31]: sounds good

**Review by lianxmfor — APPROVED** [review]
lgtm

**Pull request merged**